users-backend/users/internal_api/tests/test_views.py

```diff
@@ -1,30 +0,0 @@
-from pythonit_toolkit.headers import SERVICE_JWT_HEADER
```

Comment:
removed those tests for now... for sure I still want to add a generic validation that you can't call internal-api without the token, so they hopefully will be back